### PR TITLE
openshift/os: Use latest cosa

### PR DESCRIPTION
**Pull request by cgwalters**
It works now.

### CODE REVIEW

File: ci-operator/config/openshift/os/openshift-os-master.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ base_images:
   coreos_coreos-assembler_49:
```
Comment (bgilbert) [inline ci-operator/config/openshift/os/openshift-os-master.yaml:2]: Should we go back to calling this `_latest`?

Comment (cgwalters) [reply ci-operator/config/openshift/os/openshift-os-master.yaml]: Yep see the `latest` changes below

**Comment by jlebon** [comment]
I think there's also the 4.9 variant we'll want to change, but that can be done as a follow-up.
/lgtm

**Comment by jlebon** [comment]
> I think there's also the 4.9 variant we'll want to change, but that can be done as a follow-up.
> /lgtm

https://github.com/openshift/release/pull/20572

**Pull request merged**